packages/runtime/src/useCases/consumption/attributes/CreateAndShareRelationshipAttribute.ts

```diff
@@ -23,6 +23,10 @@ export interface CreateAndShareRelationshipAttributeRequest {
         metadata?: Record<string, any>;
         expiresAt?: ISO8601DateTimeString;
     };
+    requestItemMetadata?: {
+        title?: string;
+        description?: string;
+    };
```

Comment:
What about the `metadata` property? There is also `requireManualDecision`. I'm not sure if the acceptance can be automated, but probably this is somehow possible, so maybe we could also provide to configure that.

Reply:
good catch, I wasn't aware, that the RequestItem also has metadata